client/signup/config/flows.js

```diff
@@ -118,7 +119,21 @@ function filterDesignTypeInFlow( flowName, flow ) {
  * @return {string}          New flow name.
  */
 function filterFlowName( flowName ) {
-	// do nothing. No flows to filter at the moment.
+	if ( user.get() ) {
```

Comment:
Could be shortened to `if ( user.get() && flowName === 'user-first' )`, but understand if you're leaving it this way for the comments.

Reply:
I did it this way for a clean if/else on just the user logged in case, so the else block below is just for not logged in.